.ci/travis-after-success.sh

```diff
@@ -2,6 +2,6 @@
 
 set -e -x
 
-if [[ -z ${TOXENV} ]]; then
-    codecov -v
+if ! [[ -n ${TOXENV} ]] && ! [[ ${BIGCHAINDB_CI_ABCI} == 'enable' ]] && ! [[ ${BIGCHAINDB_ACCEPTANCE_TEST} == 'enable' ]]; then
```

Comment:
The expression would be a little more readable if you avoid double negation by using `-z` instead of `-n` and `!=` instead of `==`.

Reply:
ah, I see! Thanks for the input